src/Compilers/CSharp/Test/EndToEnd/EndToEndTests.cs

```diff
@@ -721,5 +721,83 @@ class XAttribute : System.Attribute { }
             Assert.Collection(runResult.TrackedSteps["result_ForAttributeWithMetadataName"],
                 step => Assert.True(step.Outputs.Single().Value is ClassDeclarationSyntax { Identifier.ValueText: "C1" }));
         }
+
+        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/70381")]
+        public void ExtremelyNestedCollectionExpressionDoesNotOverflow_1()
```

Comment:
>ExtremelyNestedCollectionExpressionDoesNotOverflow_1 [](http://example.com/codeflow?start=20&length=52)

Should these tests be moved elsewhere since they do not test actual stack limits and since they don't appear to be expensive.